runtime/tools/python/ttrt/__init__.py

```diff
@@ -85,6 +85,12 @@ def main():
         default=0,
         help="Seed for random number generator",
     )
+    run_parser.add_argument(
+        "--runtime-type",
+        default="ttnn",
+        choices=["ttnn", "ttmetal"],
+        help="Backend runtime type",
+    )
```

Comment:
can you add this option in the perf parser as well? in the future, I'll merge both these paths, but for the time being, they are separate. Also update /docs ttrt with a sample command using this option

Reply:
Removed this argument and added an API that dynamically sets the runtime type based on the binary file identifier

Reply:
even better :)